internal/controller/vrg_volrep.go

```diff
@@ -645,13 +644,11 @@ func (v *VRGInstance) uploadPVandPVCtoS3Stores(pvc *corev1.PersistentVolumeClaim
 	}
 
 	if err := v.addArchivedAnnotationForPVC(pvc, log); err != nil {
-		msg := fmt.Sprintf("failed to add archived annotation for PVC (%s/%s) with error (%v)",
-			pvc.Namespace, pvc.Name, err)
-		v.log.Info(msg)
+		msg := fmt.Sprintf("failed to add archived annotation: %s", err)
 		v.updatePVCClusterDataProtectedCondition(pvc.Namespace, pvc.Name,
 			VRGConditionReasonClusterDataAnnotationFailed, msg)
 
-		return errors.New(msg)
+		return fmt.Errorf("failed to add archived annotation for PVC (%s/%s): %w", pvc.Namespace, pvc.Name, err)
```

Comment:
Repeated string. Maybe just do this:
```
return fmt.Errorf("%s - PVC (%s/%s): %w", msg, pvc.Namespace, pvc.Name)
```

Reply:
The message is not the same - we don't have the pvc namespace/name in msg, and it is not needed for the condition update, since it is displayed under the protected pvc.

Reply:
Look again at my suggested line. It has everything you needed including `msg` from line 647